components/calculator/RCWLinkInfo.tsx

```diff
@@ -0,0 +1,118 @@
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import {
+  Card, CardContent, Dialog, Grid, IconButton, Typography,
+} from '@mui/material';
+import { Box } from '@mui/system';
+import MuiMarkdown from 'mui-markdown';
+import React from 'react';
+
+import theme from '../../styles/themes/theme.tsx';
+import { CalculatorButton } from '../helper/EligibilityButton.tsx';
+
+const details1 = '**RCW** stands for “Revised Code of Washington,” which is the collection of all permanent laws. ';
```

Comment:
Thanks for the suggestion, I would say no because we are using Sanity to hold all of the calculator content, however since we are just trying to do this one time and it will not be needing further changes, it is not worth the trouble of updating the Sanity repo schemas, Sanity, and then also the code just for a one time change.